app/(editor)/editor/page.tsx

```diff
@@ -86,6 +95,65 @@ async function getStoryData({ path }: PageProps['searchParams']): Promise<ISbRes
 
 };
 
+/**
+ * Get a list of stories that are of component sbStoryMvp in reverse chronological order.
+ */
+async function getStoryList({ path }: PageProps['searchParams']) {
```

Comment:
Adding duplicative or page-specific code to the editor is a code smell for me. I'd rather move this deeper into the component stack but before I make any suggestions can you tell me why you chose to go this way and what issues you ran into?

Reply:
@sherakama No specific reason here. I wanted to do server side fetching so just putting it where we're already fetching story data and in case we need it available for other pages in the future (e.g., initiative and theme pages might need their own story list), but definitely open to doing it in a more maintainable way 😄

Reply:
Ok cool. This will be a good exercise to dig deeper into server/client component interweaving. Have a read through https://nextjs.org/docs/app/building-your-application/rendering/composition-patterns#interleaving-server-and-client-components and explore a bit more then let's reconnect to see what blockers we hit.

Reply:
Thank you - I'll do that! 😄